docs/contributing/pull-requests/branching.md

```diff
@@ -51,22 +52,20 @@ Product.
 
 Choose a **Long-Lived Feature Branch** if the feature will:
 
-- Have multiple authors
-- Consist of multiple Pull Requests, which have already been reviewed
-- Require these multiple Pull Requests from multiple authors in order to produce the testable,
-  releasable change
+- Require multiple Pull Requests to produce a testable, releasable change, and
+- It is not possible to put the changes behind a feature flag
 
 Choose a **Short-Lived Feature Branch** if the feature will:
 
-- Have a single author
-- Consist of a single Pull Request from the Feature Branch into `master` to produce the testable,
-  releasable change
+- Require a single Pull Request to produce a testable, releasable change, or
+- It is possible to put the changes behind a feature flag while multiple PRs are in flight.
 
 :::tip Still Unsure?
 
 If in doubt, lean toward creating shorter-lived feature branches directly off of `master` for each
 developer’s work, as there is overhead built in to the Long-Lived Feature Branch that can be avoided
-if smaller features can be tested and released independently.
+if smaller features can be tested and released independently, or hidden behind a feature flag until
+the changes are ready.
```

Comment:
Feels like we're duplicating the same words as on line 61.

Reply:
Agree, truncated